code/renderers/html/template/cli/ts/Button.stories.ts

```diff
@@ -1,4 +1,4 @@
-import type { Meta, StoryFn } from '@storybook/html';
+import type { Meta, StoryFn } from 'renderers/html/dist';
```

Comment:
Is `renderers` a package?

Reply:
Undoing, not sure how that ended up here.